views/tos.jsx

```diff
@@ -0,0 +1,42 @@
+const React = require('react')
+const Layout = require('./shared/layout')
+
+function TosSection() {
+  return (
+    <div className="tos-section">
+      
+      <h1>Terms & Conditions</h1>
+      <h3>1. License Grant</h3>
+      <p>We grant you a worldwide, non-exclusive and non-transferable right to use our services.</p>
+
+      <h3>2. Support</h3>
+      <p>If you are in need of support, contact us at support@morehumaninternet.org. 
```

Comment:
Please convert the email to a mailto link

Reply:
mailto link added